Dockerfile

```diff
@@ -1,28 +1,41 @@
 FROM python:3.8 as boefjes-requirements
 
-COPY nl-kat-boefjes/boefjes ./boefjes
+COPY boefjes ./boefjes
+COPY requirements.txt ./boefjes
 
 # The echo since cat does not add a newline
 RUN find ./boefjes -name 'requirements.txt' -execdir sh -c "cat {} && echo" \; | sort -u > /tmp/boefjes-requirements.txt
 
+FROM python:3.8
 
+ARG USER_UID=1000
+ARG USER_GID=1000
 
-FROM python:3.8 as dev
+ENTRYPOINT ["/app/boefjes/entrypoint.sh"]
+
+RUN groupadd --gid $USER_GID nonroot
+RUN adduser --disabled-password --gecos '' --uid $USER_UID --gid $USER_GID nonroot
 
 WORKDIR /app/boefjes
+ENV PATH=/home/nonroot/.local/bin:${PATH}
 
-COPY nl-kat-octopoes/ /app/octopoes
-RUN pip install /app/octopoes
+ARG ENVIRONMENT
 
 COPY --from=boefjes-requirements /tmp/boefjes-requirements.txt /tmp/boefjes-requirements.txt
-RUN pip install -r /tmp/boefjes-requirements.txt
+COPY requirements-dev.txt .
 
-COPY nl-kat-boefjes/requirements-dev.txt .
-RUN pip install -r requirements-dev.txt
+RUN --mount=type=cache,target=/root/.cache --mount=type=secret,id=github_token \
+    git config --global url."https://`cat /run/secrets/github_token`@github.com/".insteadOf "ssh://git@github.com/" \
```

Comment:
This should also just use the public https repos without secrets..

Reply:
Better now?